src/python/gem5/components/boards/kernel_disk_workload.py

```diff
@@ -259,32 +259,3 @@ def _set_readfile_contents(self, readfile_contents: str) -> None:
         file = open(self.readfile, "w+")
         file.write(readfile_contents)
         file.close()
-
```

Comment:
This code generalized your (great) idea. I assume it was a rebase error to delete it.